docs/sorting.md

```diff
@@ -0,0 +1,218 @@
+# Sorting
+
+Note that bed fusion merges by default. So for the sorting examples we will not merge the bed regions, by using the flag `--no-merge`, to highlight the differences between the sorting methods.
+
+Sorting is always done after merging, so the resulting output will be sorted in the preferred format regardless if the bed file is merged or not.
+
+Example bed file `examples/sort-test.bed`:
+
+``` text
+2	12	13	1	C
+Y	10	11	1	A
+1	8	9	-1	B
+10	12	13	1	D
+GL000209.1	10	11	1	A
+1	10	11	-1	A
+1	12	13	1	A
+X	10	11	1	A
+1	10	11	1	A
+1	10	11	-1	B
+MT	10	11	1	A
+```
+
+## Lexicographic Sorting
+
+Lexicographic sorting is the default sorting method.
```

Comment:
```suggestion
```

Reply:
I think I want to double down on this information, so I'm keeping this sentence even if we are also adding it to the title :)

Reply:
hm, then I would rather not add it to the title. I would understand if you repeated it in a different section but having the same information twice in two consecutive lines seems a bit much 😬